Tests/test_map.gd

```diff
@@ -8,7 +8,9 @@ func test_map_gen():
 	var expected_null_array: Array[Array] = [[0,0], [0,1], [0,3],[0,4],[1,0],[1,4],[3,0],[3,4],[4,0],[4,1],[4,3],[4,4]]
 	var expected_exists_array: Array[Array] = [[0,2],[1,1],[1,2],[1,3],[2,0],[2,1],[2,2],[2,3],[2,4],[3,1],[3,2],[3,3],[4,2]]
 	for couple: Array[int] in expected_null_array:
-		assert_null(test_map.rooms[couple[0]][couple[1]], "expected null at"+str(couple))
-		
+		var _room = test_map.rooms[couple[0]][couple[1]]
+		assert_null(test_map.rooms[couple[0]][couple[1]], "Expected null at %s but got %s" % [couple, _room])
```

Comment:
since you have the `_room` variable defined, you can use that instead of `test_map.rooms[couple[0]][couple[1]]` in the assert